src/main/java/manageezpz/logic/parser/AddTodoTaskCommandParser.java

```diff
@@ -0,0 +1,43 @@
+package manageezpz.logic.parser;
+
+import static manageezpz.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static manageezpz.logic.parser.CliSyntax.PREFIX_DESCRIPTION;
+import static manageezpz.logic.parser.CliSyntax.PREFIX_TODO;
+
+import java.util.stream.Stream;
+
+import manageezpz.logic.commands.AddTodoTaskCommand;
+import manageezpz.logic.parser.exceptions.ParseException;
+import manageezpz.model.task.Description;
+import manageezpz.model.task.Todo;
+
+public class AddTodoTaskCommandParser implements Parser<AddTodoTaskCommand> {
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddEmployeeCommand
+     * and returns an AddEmployeeCommand object for execution.
+     * @throws ParseException if the user input does not conform the expected format
+     */
+    public AddTodoTaskCommand parse(String args) throws ParseException {
+        ArgumentMultimap argMultimapTodo =
+                ArgumentTokenizer.tokenize(args, PREFIX_TODO, PREFIX_DESCRIPTION);
+
+        if (!arePrefixesPresent(argMultimapTodo, PREFIX_TODO, PREFIX_DESCRIPTION)
```

Comment:
Try splitting the boolean expression into local constants instead. It'll definitely make it more readable 👍